src/gui/dialogs/addon/manager.cpp

```diff
@@ -391,6 +396,50 @@ void addon_manager::pre_show(window& window)
 	connect_signal_notify_modified(type_filter,
 		std::bind(&addon_manager::apply_filters, this));
 
+	// Language filter
+	// Clear current language filter vector, lest all entries are shown twice
+	addon_manager::language_filter_types_.clear();
+	addon_manager::language_filter_types_.push_back(std::pair<int, std::string>(0, "en_GB"));
+
+	// Prepare shown languages, source all available languages from the addons themselves
+	std::vector<std::string> languages_available;
+	for(const auto& a : addons_) {
+		for (const auto& b : a.second.locales) {
+			if (!(b.rfind("en_GB", 0) == 0)) {
+				languages_available.push_back(b);
+			}
+		}
+	}
+	std::sort(languages_available.begin(), languages_available.end());
+	auto last = std::unique(languages_available.begin(),languages_available.end());
+	languages_available.erase(last, languages_available.end());
+	// Erase pt value, since otherwise Portugues do Brasil will be shown twice in the language list
+	(void)std::remove(languages_available.begin(), languages_available.end(), "pt");
```

Comment:
We don't need `(void)` here. `std::remove` is not marked `[[nodiscard]]`.

Reply:
The two windows CI tests on the very first iteration I submitted failed because of that error, thats why I added it

Reply:
I think the issue here is that `remove` doesn't resize the list, so even if it's not marked `[[nodiscard]]` it probably should be. The idiomatic usage of `remove` is like this:

```cpp
container.erase(std::remove(container.begin(), container.end(), value), container.end())
```

So, following that is probably the correct fix for the CI error. (You could also split it into two lines by assigning the result of `remove` to a variable, if you wanted to.)

Reply:
Thx for the input, it should now be properly handled :ok_hand: